js/main.js

```diff
@@ -62,6 +64,68 @@ function renderResults(name, street, city, state, zip) {
   $info.textContent = city + ', ' + state + ', ' + zip;
   initialDiv.appendChild($info);
 
+  var $url = document.createElement('a');
+  if (url !== null) {
+    $url.setAttribute('href', url);
+    $url.textContent = 'Website: ' + url;
+  } else {
+    $url.textContent = 'No Website Available';
+  }
+  initialDiv.appendChild($url);
+  $url.className = 'view hidden';
+
+  function titleCase(string) {
+    string = string.toLowerCase().split(' ');
+    for (var i = 0; i < string.length; i++) {
+      string[i] = string[i].charAt(0).toUpperCase() + string[i].slice(1);
+    }
+    return string.join(' ');
+  }
+
+  var $type = document.createElement('p');
+  initialDiv.appendChild($type);
+  $type.className = 'view hidden';
+
+  var $span1 = document.createElement('span');
+  $span1.className = 'underline';
+  $span1.textContent = 'Brewery Type:';
+  $type.appendChild($span1);
+
+  var $span2 = document.createElement('span');
+  $span2.textContent = ' ' + titleCase(type);
+  $type.appendChild($span2);
+
+  var $button = document.createElement('button');
+  $button.className = 'dots-button';
+  initialDiv.appendChild($button);
+  $button.addEventListener('click', moreInfo);
+
+  var $icon = document.createElement('i');
+  $icon.className = 'fas fa-ellipsis fa-2x fa-icon';
+  $button.appendChild($icon);
+
+  function moreInfo(event) {
+    if ($url.className === 'view hidden') {
+      $url.className = 'view';
+    } else if ($url.className === 'view') {
+      $url.className = 'view hidden';
+    }
+    if ($type.className === 'view hidden') {
+      $type.className = 'view';
+    } else if ($type.className === 'view') {
+      $type.className = 'view hidden';
+    }
+    if ($icon.className === 'fas fa-ellipsis fa-2x fa-icon') {
+      $icon.className = 'fa-solid fa-caret-up fa-2x fa-icon';
+    } else if ($icon.className === 'fa-solid fa-caret-up fa-2x fa-icon') {
+      $icon.className = 'fas fa-ellipsis fa-2x fa-icon';
+    }
+    if (initialDiv.className === 'white-box white-box-dimensions') {
+      initialDiv.className = 'white-box new-dimensions';
+    } else if (initialDiv.className === 'white-box new-dimensions') {
+      initialDiv.className = 'white-box white-box-dimensions';
+    }
+  }
```

Comment:
Why is this function definition inside of another function definition? You should move this to the global scope